include/votca/xtp/ERIs.h

```diff
@@ -21,8 +21,9 @@
 #ifndef VOTCA_XTP_ERIS_H
 #define VOTCA_XTP_ERIS_H
 
-#include <votca/xtp/fourcenter.h>
-#include <votca/xtp/threecenter.h>
+// Local VOTCA includes
+#include "votca/xtp/fourcenter.h"
+#include "votca/xtp/threecenter.h"

```

Comment:
Should you drop the `votca/xtp` prefix as this file is on the same folder?

Reply:
yes good point

Reply:
@junghans maybe we should also be more selective in which headers we install and move most to src? Any opinions?

Reply:
Yes, I am all for it.

Reply:
Sure, but I wouldn't know which ones to move? I guess for xtp we could start by moving all of them, are there any tools that are built using xtp?

Reply:
> Shouldn't you drop the `votca/xtp` prefix as this file is in the same folder?

Ok

Reply:
> Shouldn't you drop the `votca/xtp` prefix as this file is in the same folder?

From what I have read it actually appears that leaving the prefix is a safer convention.

Reply:
There are two issues, the way how headers include each other and which ones are public headers.